### PR TITLE
Update Generate-Cortx-Build-Stack with CentOS-7.9 Instructions

### PR DESCRIPTION
**Pull request by mukul-seagate11**
### Describe your changes in brief
Update Generate-Cortx-Build-Stack with CentOS-7.9 Instructions

-----
[View rendered doc/community-build/Generate-Cortx-Build-Stack.md](https://github.com/Seagate/cortx/blob/mukul-seagate11-patch-5/doc/community-build/Generate-Cortx-Build-Stack.md)

### CODE REVIEW

File: doc/community-build/Generate-Cortx-Build-Stack.md [hunk 1]
```diff
@@ -11,6 +10,8 @@ To know about various CORTX components, see [CORTX Components guide](https://git
 
 ## Procedure
 
+#### **For CentOS 7.8**
```
Comment (shailesh-vaidya) [inline doc/community-build/Generate-Cortx-Build-Stack.md:13]: I guess the steps for `CentOS 7.8` and `CentOS 7.9` are exactly the same only the image tag is different. Maybe we can only mention using the appropriate tag as per the OS required.

File: doc/community-build/Generate-Cortx-Build-Stack.md [hunk 2]
```diff
@@ -66,87 +59,110 @@ To know about various CORTX components, see [CORTX Components guide](https://git
...
-    echo "3.10.0-1127.el7.x86_64"
-else
-    echo "$(uname.ori $1)"
-fi
-```
-7. Finally, make it executable by running: `chmod +x /bin/uname`
+15. Deploy the packages generated to create CORTX cluster using the instruction provided in [Deploy Cortx Build Stack guide](ProvisionReleaseBuild.md).
 
 
 ## Tested by:
```
Comment (shailesh-vaidya) [inline doc/community-build/Generate-Cortx-Build-Stack.md:164]: `Tested by ` section is added in code-block

File: doc/community-build/Generate-Cortx-Build-Stack.md [hunk 3]
```diff
@@ -66,87 +59,110 @@ To know about various CORTX components, see [CORTX Components guide](https://git
...
+    cortx-ha: generate cortx-ha packages.
+    cortx-management-portal: generate cortx-management-portal packages.
+    cortx-manager: generate cortx-manager packages.
+    cortx-monitor: generate cortx-monitor packages.
+    cortx-posix: generate cortx-posix (NFS) packages.
+    cortx-prvsnr: generate cortx-prvsnr packages.
+    iso_generation: generate ISO file from release build.
+    ```   
 
-You might get an error message about missing `kernel-devel` package when building the CORTX packages; sample errors:
```
Comment (shailesh-vaidya) [inline doc/community-build/Generate-Cortx-Build-Stack.md:118]: I feel it's good to have a troubleshooting section. Any specific reason to remove it.

**Review by shailesh-vaidya — CHANGES_REQUESTED** [review]

**Comment by mukul-seagate11** [comment]
Will raise new PR due to some conflicts

**Pull request closed without merging**